src/test/java/org/assertj/core/api/abstract_/AbstractAssert_satisfies_with_Consumers_Test.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2021 the original author or authors.
+ */
+package org.assertj.core.api.abstract_;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.api.BDDAssertions.thenIllegalArgumentException;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+
+import java.util.function.Consumer;
+
+import org.assertj.core.test.Jedi;
+import org.junit.jupiter.api.Test;
+
+class AbstractAssert_satisfies_with_Consumers_Test {
```

Comment:
use `hasMessageNotContaining​` and `hasMessageContaining​` whenever possible in these tests